pkgs/applications/misc/pomodoro/default.nix

```
@@ -0,0 +1,26 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+let
+  pname = "pomodoro";
+  version = "0.1.0";
+in rustPlatform.buildRustPackage {
+  inherit pname;
+  inherit version;
+  src = fetchFromGitHub {
+    # Anna Aurora forked SanderJSA's repo to add git version tags. SanderJSA didn't respond when Anna Aurora requested dem to add git version tags.
+    owner = "auroraanna";
+    repo = pname;
```

```suggestion
    owner = "SanderJSA";
    repo = "Pomodoro";
```
Using a fork just for a git tag is not worth it since we might miss updates from the upstream repo if it is not updated.

---

upstream seems to have abandoned the project

---

You're right @SuperSandro2000, kind of didn't of making it unstable since there is a version number in the `Cargo.toml`.